src/App.vue

```diff
@@ -1,12 +1,21 @@
 <template>
   <div id="app">
     <router-view/>
+    <page-home/>
+    <app-modal/>
   </div>
 </template>
 
 <script>
+import AppModal from './components/AppModal'
+import PageHome from './components/PageHome'
```

Comment:
same

Reply:
So, `PageHome` should be added to the router?